src/libraries/System.Security.Cryptography.X509Certificates/tests/ExportTests.cs

```diff
@@ -110,7 +110,7 @@ public static void ExportAsPfxWithPrivateKeyVerifyPassword()
             {
                 Assert.True(cert.HasPrivateKey, "cert.HasPrivateKey");
 
-                const string password = "Cotton";
+                const string password = "password";
```

Comment:
Please revert these changes, and suppress them as appropriate.  (Some tests need to not use the password "password")